src/Bridges/Constraint/indicator.jl

```diff
@@ -51,3 +51,89 @@ function concrete_bridge_type(::Type{<:IndicatorActiveOnFalseBridge},
                               ::Type{MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO, S}}) where {F<:MOI.VectorAffineFunction, S<:MOI.AbstractScalarSet}
     return IndicatorActiveOnFalseBridge{Float64, F, S}
 end
+
+## Indicator constraint by SOS1 bridge
+
+"""
```

Comment:
I would move it to another file `indicator_sos1.jl`

Reply:
done in last commit